src/main/java/salesforce/entities/TaskUi.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.entities;
+
+import salesforce.entities.constants.TaskConstant;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * Manages Task information.
+ *
+ * @author Alan Escalera.
+ * @version 1.0 24 March 2020.
+ */
+public class TaskUi {
```

Comment:
What is the rason for this class? or it is Taks entity?, it would be better Task only

Reply:
x2